lib/rubocop/cop/layout/hash_alignment.rb

```diff
@@ -248,10 +248,17 @@ def check_pairs(node)
           add_offences
         end
 
+        def specific_message(format)
+          messages = { KeyAlignment => 'Align the keys of a hash literal if they span more than one line.',
+                       SeparatorAlignment => 'Align the separator of a hash literal if they span more than one line.',
+                       TableAlignment => 'Align the keys and values of a hash literal if they span more than one line.'}
+          return messages[format]
+        end
+
         def add_offences
           _format, offences = offences_by.min_by { |_, v| v.length }
```

Comment:
Now that `_format` is no longer unused, better drop the underscore from its name.